src/main/com/aca/homework/week9/calculator/Operation.java

```diff
@@ -0,0 +1,43 @@
+package com.aca.homework.week9.calculator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public enum Operation {
+    ADDITION {
+        @Override
+        public String performOperation(int num1, int num2) {
+            return String.valueOf(num1 + num2);
+        }
+    },
+    SUBTRACTION {
+        @Override
+        public String performOperation(int num1, int num2) {
+            return String.valueOf(num1 - num2);
+        }
+    },
+    MULTIPLICATION {
+        @Override
+        public String performOperation(int num1, int num2) {
+            return String.valueOf(num1 * num2);
+        }
+    },
+    DIVISION {
+        @Override
+        public String performOperation(int num1, int num2) {
+            return String.valueOf((double) num1 / num2);
+        }
+    };
+
+    private static final Map<Character, Operation> map = Map.of('+', Operation.ADDITION, '-', Operation.SUBTRACTION, '*', Operation.MULTIPLICATION, '/', Operation.DIVISION);
```

Comment:
just pass a char as a constructor argument. there is no need to use a map

Reply:
don't I need a map for implementing the ``Operation of()`` and ``boolean isOperation()`` methods?

Reply:
please pass + to constructor for ADDITION value,  to SUBTRACTION and so on. the using enum.values() filter and find the enum for the provided char